docs/README.md

```diff
@@ -51,23 +51,7 @@ An example of the body sent in the checkout post request is:
 ```
 
 ### Provider to VTEX
-In this example, we focus on a sync request done by the provider to the checkout. In that case, it's also possible to implement a parser in order to put the payload in the format that the checkout expects. One can find an example of the expected format for a sync request:
-
-```json
-[
-    {
-        "id": "0",
-        "taxes": [
-            {
-                "name": "TAX 1",
-                "value": 100
-            }
-        ]
-    }
-]
-```
-
-In case of an async request, the expected format is:
+In this example, we focus on a sync request done by the provider to the checkout. In that case, it's also possible to implement a parser in order to put the payload in the format that the checkout expects. One can find an example of the expected format:
```

Comment:
```suggestion
In this example, we focus on a synchronous (??) request made by the provider to Checkout. It's also possible to implement a parser to map the received payload to the format expected by Checkout. One can find an example of the expected format:
```

`One can find an example of the expected format` -> não entendi bem se isso é um example recebido ou parseado